frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx

```diff
@@ -99,34 +164,79 @@ class AddSubCategory extends React.Component<
             <Row>
               <Col md={12}>
                 <Spin tip="Loading..." spinning={this.state.isLoading}>
-                  <Form size="large" onFinish={this.onFinish}>
-                    <Title level={3}>Category</Title>
-                    <Text>Select a category</Text>
-                    <Form.Item name="category" className={mainStyles.formItem}>
-                      <Select
-                        className={mainStyles.formSelect}
-                        placeholder="Please select a category"
+                  {this.state.subcategory != null && (
+                    <Form
```

Comment:
Shall we reuse the form too?

Reply:
Didn’t get you @jayasanka-sack

Reply:
There are two forms defined in the component. 

Let's try to achieve this using a single form

Reply:
two different functions using two forms and I can’t set initial values with conditional rendering because it's works only for elements and titles are also different.